index.js

```diff
@@ -36,6 +37,14 @@ function init() {
     return date.toISOString();
   }
 
+  var error = function(){
+    return function(){
+        let args = Array.prototype.slice.call(arguments);
+        args.unshift('err');
+        this.log.apply(this, args);
+    }	
+  }
+  
```

Comment:
Why do you wrap it in another function? Just leave it like this:
```js
function error() {
  let args = Array.prototype.slice.call(arguments);
  args.unshift('err');
  this.log.apply(this, args);
}
```

Reply:
@mbrandau .. i tried it without wrapping ... build(npm run test) does  not pass ... 
because for all the keys in the return function ... the value should be something that returns a function .. :)

Reply:
Follow my comment and the one below and have a look at `test/index.js`. You'll find a levels array somewhere. Change `err` to `error` and we should be fine.

Reply:
ook ... i will look into it ... :)